collapsing/index.md

```diff
@@ -0,0 +1,44 @@
+# Collapsing Towers for Fun and Profit
```

Comment:
Can you make the title collapsing towers of interpreters and link it from index page?

Reply:
no whimsy allowed :pensive:

Reply:
too cliché...